hvac/api/secrets_engines/identity/mfa/mfa_method_mixin.py

```diff
@@ -0,0 +1,92 @@
+from abc import ABCMeta, abstractmethod
+
+from hvac import utils
+from hvac.api.vault_api_base import VaultApiBase
+from hvac.constants.identity import DEFAULT_MOUNT_POINT
+
+
+class MfaMethodMixin(VaultApiBase, metaclass=ABCMeta):
```

Comment:
How do you feel about the naming of methods for the mfa methods?
You end up using this:
- `client.secrets.identity.mfa.okta.create(...)`
- `client.secrets.identity.mfa.okta.update(...)`
- `client.secrets.identity.mfa.okta.read(...)`
- `client.secrets.identity.mfa.okta.delete(...)`
- `client.secrets.identity.mfa.okta.list(...)`


Or should this instead be:
- `client.secrets.identity.mfa.okta.create_mfa_method(...)`
- `client.secrets.identity.mfa.okta.update_mfa_method(...)`
- `client.secrets.identity.mfa.okta.read_mfa_method(...)`
- `client.secrets.identity.mfa.okta.delete_mfa_method(...)`
- `client.secrets.identity.mfa.okta.list_mfa_methods(...)`

And should the totp-specific methods be renamed?
- `client.secrets.identity.mfa.totp.generate(...)` -> `client.secrets.identity.mfa.totp.generate_mfa_secret(...)`
- `client.secrets.identity.mfa.totp.admin_generate(...)` -> `client.secrets.identity.mfa.totp.admin_generate_mfa_secret(...)`
- `client.secrets.identity.mfa.totp.admin_destroy(...)` -> `client.secrets.identity.mfa.totp.admin_destroy_mfa_secret(...)`